docker-compose.yml

```diff
@@ -830,6 +807,7 @@ services:
     command: dotnet /build/bin/Debug/_build.dll RunDebuggerIntegrationTests
     volumes:
       - ./:/project
+      - /var/run/docker.sock:/var/run/docker.sock
```

Comment:
So I guess before we didn't need socket access, but I started getting a ton of connection issues and this seemed to resolve that.